### ISSUE TITLE
hackage release doesn't allow ghc-9.2 and newer libs

### ISSUE THREAD

**Issue by l29ah** [zmthy/http-media#41]
unlike git master
```
setup: Encountered missing or private dependencies:
QuickCheck >=2.8 && <2.14,
base >=4.8 && <4.13,
test-framework >=0.8 && <0.9,
test-framework-quickcheck2 >=0.3 && <0.4
```

**Comment by zmthy** [comment]
Is this in the context of pulling the Hackage package and then building the tests? Metadata revisions have been applied to the library, but I don't think they were applied to the test-suite. I think I can push another revision that bumps the test-suite dependencies too, would that solve your problem?

**Comment by l29ah** [comment]
Seems so.

**Comment by zmthy** [comment]
Sorry for the wait, but it turned out that you can't revise the dependencies of test suites. This should be fixed now by using the new version 0.8.1.0.
